### ISSUE TITLE
Most recent commit completely breaks this tool

### ISSUE THREAD

**Issue by Samathy** [ohdatboi/profdump#5]
At the time of writing, commit 0e2564f completely removes `int main()` from `source/app.d`, thus making the tool unusable.
The previous commit ( e1910d1 ) still works fine though.

To use this tool, clone the repo, `git checkout e19101d` and `dub build` as you'd expect.

Is this project dead and the latest commit was intended as an indication of the deadness?

Thanks :)

**Comment by AntonMeep** [comment]
Thank you for the report!

Since this was more than a year ago, I don't really remember why did I do that. Probably, I just wanted to move `main()` function inside of the profdump package but never got back to it 🤷 

The project is not dead. Even though I don't actively develop it now, it worked fine for me last time I needed it (no idea how so when there's no `main()` but still) and the format of the profiler's output didn't seem to change since.

I will fix the project next week and tag the final release. This is all I can do because I don't actively develop using the D programming language, just trying support all of my old stuff

**Comment by Samathy** [comment]
Yes the profile output format doesnt seem to have changed, so thats a perfect solution, thanks a lot!

**Comment by aminya** [comment]
I made a pull request that fixes this issue @AntonMeep
